src/components/Footer.tsx

```diff
@@ -0,0 +1,27 @@
+import styled from 'styled-components';
+import { SheepLogo } from './SheepLogo';
+
+const StyledFooter = styled.footer`
+    padding: 20px 20px;
+    display: flex;
+    justify-content: start;
+    max-height: 60px;
+`;
+
+const StyledFooterText = styled.span`
+    font-weight: 400;
```

Comment:
This is default weight. Unnecessary here.

Reply:
done